incubator/metrics-http/src/main/java/io/aklivity/zilla/runtime/metrics/http/internal/HttpCollectorContext.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2021-2022 Aklivity Inc
+ *
+ * Licensed under the Aklivity Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ *   https://www.aklivity.io/aklivity-community-license/
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package io.aklivity.zilla.runtime.metrics.http.internal;
+
+import io.aklivity.zilla.runtime.engine.metrics.CollectorContext;
+public class HttpCollectorContext implements CollectorContext
+{
+}
```

Comment:
This class is not necessary, please remove it.
Typically only the engine provides an implementation of the collector context interface.